### PR TITLE
Issue 24

### PR DESCRIPTION
**Pull request by james-herbstritt**

### CODE REVIEW

File: src/app/components/video/video.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<video autoplay loop>
+    <source
+            src="../../assets/mp4"
+            type="video/mp4">
+            Your browser does not support this version.
+</video>
```
Comment (mmgoodnow) [inline src/app/components/video/video.component.html]: this file needs to be prettiered

File: src/app/components/debug-window/debug-window.component.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="mb-1 mt-1">
+<button class="btn btn-block">
+    Clamp Arm?
+</button>
+</div>
+<div>
+<button class="btn btn-block">
+    Clamp Leg???
+</button>
+</div>
```
Comment (mmgoodnow) [inline src/app/components/debug-window/debug-window.component.html]: this file needs to be prettiered

Comment (mmgoodnow) [reply src/app/components/debug-window/debug-window.component.html]: also you can put both buttons in a single div with .d-block

File: src/app/components/debug-page/debug-page.component.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+             <app-video></app-video>
+        </div>
+    </div>
+
+   <div class="row">
+       <div class="col-6">
+            <app-video></app-video>
+       </div>
+
+       <div class="col-6" style="border: solid #000">
```
Comment (mmgoodnow) [inline src/app/components/debug-page/debug-page.component.html]: define styles in the scss file

**Review by mmgoodnow — COMMENTED** [review]
I think we should add HTML to the lint-staged prettier config in package.json. Other than that and the inline styles it's looking good

File: src/app/components/debug-page/debug-page.component.html (hunk 3 above)
Comment (mmgoodnow) [reply src/app/components/debug-page/debug-page.component.html]: you can also probably use a bootstrap-defined color and probably don't need the hash

File: src/app/components/video/video.component.html (hunk 1 above)
Comment (mmgoodnow) [reply src/app/components/video/video.component.html]: Also is this valid HTML? I don't see a `</source>` tag

Comment (james-herbstritt) [reply src/app/components/video/video.component.html]: https://www.w3schools.com/tags/tag_video.asp yup.

Comment (mmgoodnow) [reply src/app/components/video/video.component.html]: oh lol
the tag I was talking about didn't render in the markdown bc markdown tries to read html tags. I meant `</source>`

Comment (mmgoodnow) [reply src/app/components/video/video.component.html]: but interesting, what's the difference between using a source html element and using the src attribute on the video element?

Comment (james-herbstritt) [reply src/app/components/video/video.component.html]: you can specify type.

**Review by mmgoodnow — APPROVED** [review]
Looks good to me 🚀

**Review by bochola — APPROVED** [review]
I think I'm following along here, and tho I don't 100% fully understand everything, from what I gather its just another display page that shows the debug info right?

**Pull request merged**